.rubocop_todo.yml

```diff
@@ -0,0 +1,62 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2018-03-03 10:25:38 -0500 using RuboCop version 0.52.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 11
+# Configuration parameters: CountComments, ExcludedMethods.
+Metrics/BlockLength:
```

Comment:
Some of this config will need tweaking, but I think it's good to have a starting point.